staging/src/k8s.io/component-base/metrics/testutil/metrics.go

```diff
@@ -147,3 +147,35 @@ func ComputeHistogramDelta(before, after model.Samples, label model.LabelName) {
 func makeKey(a, b model.LabelValue) string {
 	return string(a) + "___" + string(b)
 }
+
+// GetMetricValuesForLabel returns value of metric for a given dimension
+func GetMetricValuesForLabel(ms Metrics, metricName, label string) map[string]int64 {
+	result := make(map[string]int64)
+	for method, samples := range ms {
+		if method != metricName {
```

Comment:
If `ms` is actually a map, this loop and `if` can be replaced with `ms[metricName]` if I'm reading it correctly.

Reply:
Good point.